README.md

```diff
@@ -56,9 +58,9 @@ If you use logback, register the OpenTelemetry logback appender in `logback-spri
 </configuration>
 ```
 
-## Log4j2
+### Log4j2
 
-If you use log4j2, register the OpenTelemetry logback appender in `log4j2.xml`:
+If you use log4j2, register the OpenTelemetry log4j2 appender in `log4j2.xml` - which you typically have in `main/resources`:
```

Comment:
To register a log4j2 appender, create a new log4j2.xml file under your project’s `main/resources` directory and copy/paste/save the following xml.

Reply:
Done in #23